compose/compose.go

```diff
@@ -572,6 +575,85 @@ func fatal(err error) {
 	os.Exit(1)
 }
 
+func makeDir(path string) error {
+	_, err := os.Stat(path)
+	if os.IsNotExist(err) {
+		if errs := os.MkdirAll(path, 0755); errs != nil {
+			fatal(errors.Errorf("Couldn't create directory %v. Error: %v", path, errs))
+		}
+	} else if err != nil {
+		fatal(errors.Errorf("Something went wrong while checking if directory %v still exists. Error: %v",
+			path, err))
+	}
+	return nil
+}
+
+func copyFile(src, dst string) error {
+	var err error
+	var srcfd *os.File
+	var dstfd *os.File
+	var srcinfo os.FileInfo
+
+	if srcfd, err = os.Open(src); err != nil {
+		fatal(errors.Errorf("Error in opening source file %v. Error: %v", src, err))
+		return err
+	}
+	defer srcfd.Close()
+
+	if dstfd, err = os.Create(dst); err != nil {
+		fatal(errors.Errorf("Error in creating destination file %v. Error: %v", dst, err))
+		return err
+	}
+	defer dstfd.Close()
+
+	if _, err = io.Copy(dstfd, srcfd); err != nil {
+		fatal(errors.Errorf("Error in copying source file %v to destination file %v. Error: %v",
+			src, dst, err))
+		return err
+	}
+	if srcinfo, err = os.Stat(src); err != nil {
+		fatal(errors.Errorf("Error in doing stat of source file %v. Error: %v", src, err))
+		return err
+	}
+	return os.Chmod(dst, srcinfo.Mode())
+}
+
+func copyDir(src string, dst string) error {
+	var err error
+	var fds []os.FileInfo
+	var srcinfo os.FileInfo
+
+	if srcinfo, err = os.Stat(src); err != nil {
+		fatal(errors.Errorf("Error in doing stat of source dir %v. Error: %v", src, err))
+		return err
+	}
+
+	if err = os.MkdirAll(dst, srcinfo.Mode()); err != nil {
+		fatal(errors.Errorf("Error in making dir %v. Error: %v", dst, err))
+		return err
+	}
+
+	if fds, err = ioutil.ReadDir(src); err != nil {
+		fatal(errors.Errorf("Error in reading source dir %v. Error: %v", src, err))
```

Comment:
Looks like the error logs in this PR is a ~little inconsistent, for ex: compare this thrown error, with [this one](https://github.com/dgraph-io/dgraph/blob/13ace774970835ddea4a0cc8a6e9728fae749103/compose/compose.go#L646) -- the format is a little different in terms of punctuation/capitalization. One option is, to keep it consistent, format the error string along the lines of [this one](https://github.com/dgraph-io/dgraph/blob/69c83890a4511677a17dcb53935fcedf12c8a875/compose/compose.go#L779)? Wdyt?

Reply:
Made error logging consistent.